src/test/java/org/mule/service/http/impl/service/server/grizzly/HttpGrizzlyServerManagerTestCase.java

```diff
@@ -103,6 +151,36 @@ private void assertProviderUsage(int adds, int removes) {
            times(removes)).removeExecutor(any());
   }
 
+  @Test
```

Comment:
Add Allure annotations (Issue and Description)

Reply:
And clarify that you want to check that a variable is properly set because we are delegating the max headers check to grizzly